core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -619,7 +619,9 @@ private[spark] object SparkConf extends Logging {
         "Not used anymore. Please use spark.shuffle.service.index.cache.size"),
```

Comment:
@vanzin, not related to this PR but do you know why it's called deprecated instead of removed config?

Reply:
Because the name of the variable hardly matters, what you want is a message that explains what's happening with that config (which is what the user sees). It was initially created for deprecated configs but works just as well for removed ones that we want to warn about.

Reply:
Ah, thanks. It might be better to rename it later then :-).